react/DiscountBadge.js

```diff
@@ -16,13 +16,15 @@ class DiscountBadge extends Component {
     const percent = this.calculateDiscountTax(listPrice, sellingPrice)
     return percent ? (
       <div className="vtex-discount-badge relative dib">
-        <div className="f7 dark-gray absolute right-0 pa2-s bg-white">
-          { label === '' && '-' }
-          <FormattedNumber value={percent} style="percent" /> { label }
+        <div className="f7 white absolute right-0 pa2-s bg-red">
```

Comment:
Did Pamela agree with this colours?

Reply:
She will check today.

Reply:
@leticiawanderley, vi agora isso, n sei usar o git :)
Mas sim,  estava discutindo com Bruno e decidimos deixar quase tudo em tons de cinza e só algumas coisas com cores, coisas que devem chamar atenção.
Já que eles já estão usando o styleguide e muitas coisas já possuem as cores, verde, vermelho e azul, vamos usar elas.

Reply:
Tá ok :) perguntei porque esse vermelho a gente usa para indicar erro no admin daqui, justamente porque chama atenção.